h/api/views.py

```diff
@@ -76,9 +76,18 @@ def index(context, request):
 @api_config(context=Root, name='search')
 def search(request):
     """Search the database for annotations matching with the given query."""
+    search_normalised_uris = request.feature('search_normalised')
```

Comment:
Oof. We have a keyword argument propagating through two layers of abstraction just to support putting it behind a feature flag? Is this a case where maybe we can just say we're special enough to warrant using `get_current_request()`? Maybe it's silly for our feature flag check to even be a request property when we could be using the query property on the `Feature` model like we do with the other models. Of course, the problem with that is that the feature flag state may depend on other properties of the request, like the admin and staff flags of the current user, so hmmmmmmmmmm